gensim/models/poincare.py

```diff
@@ -173,8 +173,34 @@ def __init__(self, train_data, size=50, alpha=0.1, negative=10, workers=1, epsil
         self._loss_grad = None
         self.build_vocab(train_data)
 
-    def build_vocab(self, relations=None, update=False):
-        """Load relations from the train data and build vocab."""
+    def build_vocab(self, relations, update=False):
+        """Build vocabulary from a relations.
+        Each relations must be a tuples of unicode strings.
+
+        Parameters
+        ----------
+        relations : list of tuples
+            List of tuples of positive examples of the form (node_1_index, node_2_index).
+        update : bool
+            If true, the new nodes in `relations` will be added to model's vocab.
```

Comment:
But `relations` doesn't contain nodes. Its description above says it contains "node indexes" (btw where does the user find those?).

Also, what happens if `False` (the default)?

Reply:
I modified the description of relations using __init__ description.
> But relations doesn't contain nodes. Its description above says it contains "node indexes" (btw where does the user find those?).

If `update=False`, the embeddings are initialized by random values.
(It means that the trained embeddings are cleaned.)
> Also, what happens if False (the default)?

Reply:
OK, I don't really understand that, but that information should appear in the documentation.

Reply:
I added update=False description in build_vocab 👍